tests/src/Functional/EventIntegrationTest.php

```diff
@@ -153,12 +157,14 @@ public function testCreateEvent(): void {
     $page->fillField('oe_sc_event_dates[0][value][time]', '02:12:22');
     $page->fillField('oe_sc_event_dates[0][end_value][date]', '2022-02-24');
     $page->fillField('oe_sc_event_dates[0][end_value][time]', '20:00:00');
+    $page->fillField('Registration URL', 'Example Event title  (1)');
     $page->pressButton('Save');
 
     $assert_session->responseContains('image-test.png');
     $assert_session->responseContains('Starter Image test');
     $assert_session->responseContains('Starter Image test alt');
     $assert_session->responseContains('Starter Image caption');
+    $assert_session->elementNotExists('css', 'div > a[href="https://europa.eu"]');
```

Comment:
We could use 'classy' theme for more precise selectors.
Then we could do this:
```suggestion
    $this->assertSame('/build/node/1', $assert_session->elementExists('css', '.field--name-oe-sc-event-registration-url > .field__item > a')->getAttribute('href'));
```
or this:
```suggestion
    $assert_session->elementExists('css', '.field--name-oe-sc-event-registration-url > .field__item > a[href="/build/node/1"]');
```
Otherwise the assertion has a limited usefulness.